src/lib/button/button.md

```diff
@@ -8,23 +8,22 @@ and accessible experience for users. A `<button>` element should be used wheneve
 is performed. An `<a>` element should be used whenever the user will _navigate_ to another view.
 
 
-There are five button variants, each applied as an attribute:
+There are seven button variants, each applied as an attribute:
 
 | Attribute            | Description                                                              |
 |----------------------|--------------------------------------------------------------------------|
-| `mat-button`         | Rectangular button w/ no elevation.                                      |
-| `mat-raised-button`  | Rectangular button w/ elevation                                          |
-| `mat-stroked-button` | Rectangular button w/ an outline border                                  |
+| `mat-button`         | Rectangular text button w/ no elevation.                                 |
+| `mat-raised-button`  | Rectangular contained button w/ elevation                                |
```

Comment:
While we're at it, the punctuation on these isn't consistent. Some descriptions end with a full-stop and some don't. Can you add full stops to all of them?